docs/proposals/podpolicy.md

```diff
@@ -0,0 +1,149 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest release of this document can be found
+[here](http://releases.k8s.io/release-1.1/docs/proposals/choosing-scheduler.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# PodPolicy and its use in choosing the scheduler in a multi-scheduler system
+
+## Introduction
+
+This document describes a new API resource, `PodPolicy`, for defining policies that apply to pods
+based on the pods' characteristics. Its initial use is to serve as the configuration for a
+new admission controller that will assign the scheduler name annotation in a
+[multi-scheduler](multiple-schedulers.md) Kubernetes system.
+
+## API
+
+```go
+// PodPolicySpec defines the configuration of the PodPolicy API resource.
+// Each rule is tested in order, and the first one that matches is applied.
```

Comment:
I think I see your point. It seems like there are two options:
(1) say that every policy is in a separate object (your second example) -- this seems like it might be the best
(2) single policy object but add an extra level to the hierarchy, so it would look something like

```
type PodPolicySpec struct {
    Policies []Policy
} 

type Policy struct {
     Rules []PodPolicyRule
}

(remainder as-is)
```
So you'd have one Policy for network-policy, one for color-policy, etc.

I think (1) is better.